docs/guides/mssqlserver/monitoring/using-prometheus-operator.md

```diff
@@ -0,0 +1,439 @@
+    matchNames:
+      - demo
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: database
+      app.kubernetes.io/instance: mssql-monitoring
+      app.kubernetes.io/managed-by: kubedb.com
+      app.kubernetes.io/name: mssqlservers.kubedb.com
+      kubedb.com/role: stats
+```
+
+Notice that the `ServiceMonitor` has label `release: prometheus` that we had specified in MSSQLServer CR.
+
+Also notice that the `ServiceMonitor` has selector which match the labels we have seen in the `mssql-monitoring-stats` service. It also, target the `metrics` port that we have seen in the stats service.
+
+## Verify Monitoring Metrics
+
+At first, let's find out the respective Prometheus pod for `prometheus-kube-prometheus-prometheus` Prometheus server.
+
+```bash
+$ kubectl get pod -n monitoring -l=app.kubernetes.io/name=prometheus
+NAME                                                 READY   STATUS    RESTARTS         AGE
+prometheus-prometheus-kube-prometheus-prometheus-0   2/2     Running   1                16d
+```
+
+Prometheus server is listening to port `9090` of `prometheus-prometheus-kube-prometheus-prometheus-0` pod. We are going to use [port forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to access Prometheus dashboard.
+
+Run following command on a separate terminal to forward the port 9090 of `prometheus-prometheus-0` pod,
+
+```bash
+$ kubectl port-forward -n monitoring prometheus-prometheus-kube-prometheus-prometheus-0 9090
+Forwarding from 127.0.0.1:9090 -> 9090
+Forwarding from [::1]:9090 -> 9090
+```
+
+Now, we can access the dashboard at `localhost:9090`. Open [http://localhost:9090](http://localhost:9090) in your browser. You should see `metrics` endpoint of `mssql-monitoring-stats` service as one of the targets.
+
+<p align="center">
+  <img alt="Prometheus Target" src="/docs/images/mssqlserver/monitoring/mssql-monitoring-targets.png" style="padding:10px">
+</p>
+
+Check the `endpoint` and `service` labels. It verifies that the target is our expected database. Now, you can view the collected metrics and create a graph from homepage of this Prometheus dashboard. You can also use this Prometheus server as data source for [Grafana](https://grafana.com/) and create beautiful dashboards with collected metrics.
+
+# Grafana Dashboards
+
+There are three dashboards to monitor Microsoft SQL Server Databases managed by KubeDB.
+
+- KubeDB / MSSQLServer / Summary: Shows overall summary of Microsoft SQL Server instance.
+- KubeDB / MSSQLServer / Pod: Shows individual pod-level information.
+- KubeDB / MSSQLServer / Database: Shows Microsoft SQL Server internal metrics for an instance.
+> Note: These dashboards are developed in Grafana version 7.5.5
+
+
+To use KubeDB `Grafana Dashboards` to monitor Microsoft SQL Server Databases managed by `KubeDB`, Check out [mssqlserver-dashboards](https://github.com/ops-center/grafana-dashboards/tree/master/mssqlserver)
```

Comment:
I don't see any images of grafana dashboard, aren't there any?

Reply:
[mssqlserver-dashboards](https://github.com/ops-center/grafana-dashboards/tree/master/mssqlserver)
here